packages/shared/src/components/pills/DappVerificationPill.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+    import { Pill } from '@bloomwalletio/ui'
+    import { localize } from '@core/i18n'
+    import { DappVerification } from '@auxiliary/wallet-connect/enums'
+
+    export let verifiedState: DappVerification
+
+    let color: 'success' | 'warning' | 'danger' = 'success'
+    $: {
+        switch (verifiedState) {
+            case DappVerification.Valid:
+                color = 'success'
+                break
+            case DappVerification.Unknown:
+                color = 'warning'
+                break
+            case DappVerification.Invalid:
+            case DappVerification.Scam:
+                color = 'danger'
+                break
+        }
+    }
```

Comment:
Do you want a constant to use across multiple files for these colours instead of these idividual switch statements?